less/index.less

```diff
@@ -49,11 +49,173 @@ table {
 }
 
 /* Set every element's box-sizing to border-box */
+
+
 * {
     box-sizing: border-box;
 }
 
-html, body {
-    height: 100%;
+html {
+	font-size:62.5%
+	
+}
+
+body {
+	background: #fdf6e4;
+	
+}
+
+.container {
+	font-size:1.6rem;
 }
 
+.container {
+	width: 880px;
+	max-width: 800px;
+	min-width: 660px;
+	margin: 0 auto;
+	border: 1px solid black;
+	.main-nav {
+		background: #2071c8;
+		border: 1px solid black;
+		height: 13vh;
+		.nav-line{
+			display: flex;
+			justify-content: center;
+			align-items: flex-end;
+			height: 13vh;
```

Comment:
you can just set it to 100% height since you already set container height to 13vh. If you have it in two separate places, than you need to remember to update it in two separate places instead of one in the future